src/prefect/runner.py

```diff
@@ -623,6 +629,31 @@ async def _get_scheduled_flow_runs(
         self._logger.debug(f"Discovered {len(scheduled_flow_runs)} scheduled_flow_runs")
         return scheduled_flow_runs
 
+    def _acquire_limit_slot(self, flow_run_id: str) -> bool:
+        """
+        Enforces flow run limit set on runner.
+
+        Returns:
+            - bool: True if a slot was acquired, False otherwise.
+        """
+        try:
+            if self._limiter:
+                self._limiter.acquire_on_behalf_of_nowait(flow_run_id)
+            return True
+        except anyio.WouldBlock:
+            self._logger.info(
+                f"Flow run limit reached; {self._limiter.borrowed_tokens} flow runs"
+                " in progress."
+            )
```

Comment:
This could be a good place to let users know how they can raise the runner limit if they need to.

Reply:
Yea good call! Will update the log

Reply:
Updated